Assets/_BForBoss/_Core/Scripts/Enemies/AgentNavigationBehaviour.cs

```diff
@@ -2,21 +2,23 @@
 using UnityEngine;
 using UnityEngine.AI;
 
-namespace Perigon.Entities
+namespace BForBoss
 {
     [RequireComponent(typeof(NavMeshAgent))]
     public class AgentNavigationBehaviour : MonoBehaviour
     {
         [SerializeField] 
```

Comment:
if we want to be safe, we can make this field have a `Min` attribute so that this value can't be less than 0.